ui-participant/src/login/RedirectFromOAuth.tsx

```diff
@@ -54,11 +54,12 @@ export const RedirectFromOAuth = () => {
 
           const email = auth.user.profile.email as string
           const accessToken = auth.user.access_token
+          const isProxy = true
```

Comment:
This was for the sake of testing the flow through the frontend, I will change this before making this an actual PR